app-backend/db/src/main/resources/migrations/V8__Add_cascading_deletion_to_user_fks.sql

```diff
@@ -0,0 +1,125 @@
+BEGIN;
+
+ALTER TABLE "access_control_rules"
+DROP CONSTRAINT "access_control_rules_created_by_fkey",
+ADD CONSTRAINT "access_control_rules_created_by_fkey" FOREIGN KEY (created_by) REFERENCES users(id) ON DELETE CASCADE;
```

Comment:
I think to prevent some lengthy downtime because of the long time required for the validation [step](https://travisofthenorth.com/blog/2017/2/2/postgres-adding-foreign-keys-with-zero-downtime) it might be better to mark these as non-valid to start with and then have another migration to set them as valid.

I don't think we have to worry about cleaning things up since this will be adding them one after the other.